packages/d3fc-series/src/canvas/multi.js

```diff
@@ -18,15 +18,21 @@ export default () => {
                 .xScale(xScale)
                 .yScale(yScale);
 
-            const adaptedDecorate = dataSeries.decorate();
-            dataSeries.decorate((c, d, i) => {
-                base.decorate()(c, data, index);
-                adaptedDecorate(c, d, i);
-            });
+            let adaptedDecorate;
+            if (dataSeries.decorate) {
+                adaptedDecorate = dataSeries.decorate();
+                dataSeries.decorate((c, d, i) => {
+                    base.decorate()(c, data, index);
+                    adaptedDecorate(c, d, i);
+                });
+            }
 
             dataSeries(seriesData);
+            base.decorate()(context, data, index);
```

Comment:
Sorry it's taken a little while to look at this. It all looks good to me apart from this line, I'm not sure why it's there?

Reply:
That was in reference to your comment about calling `base.decorate` even if `dataSeries.decorate` is undefined. Perhaps I misunderstood? I can remove the line if you'd prefer.